apps/src/lab2/views/components/guide/Guide.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import styles from './Guide.module.scss';
+
+interface GuideProps {
+  id?: string;
+  children: React.ReactNode;
+}
+
+const Guide: React.FunctionComponent<GuideProps> = ({id, children}) => {
```

Comment:
nit: mind adding a quick doc on what this component is meant to do? (since it's mostly in the styling)